todo.sh

```diff
@@ -1485,14 +1485,22 @@ note: PRIORITY must be anywhere from A to Z."
 "listaddons" )
     if [ -d "$TODO_ACTIONS_DIR" ]; then
         cd "$TODO_ACTIONS_DIR" || exit $?
+        VALID=0
         for action in *
         do
             if [ -f "$action" ] && [ -x "$action" ]; then
                 echo "$action"
+                VALID+=1
             elif [ -d "$action" ] && [ -x "$action/$action" ]; then
                 echo "$action"
+                VALID+=1
             fi
         done
+        if ! [ "$VALID" -gt 0 ]; then
+             echo "TODO: '$TODO_ACTIONS_DIR' does not contain valid actions."
+        fi 
+    else
+        [ "$TODOTXT_VERBOSE" -gt 0 ] && echo "TODO: '$TODO_ACTIONS_DIR' does not exist." 
```

Comment:
Errors should be printed to standard error, not out, and the completion already suppresses stderr output. There's already the `die()` function for that, it also exits with 1, which is nice: another indication that something's wrong.

Reply:
Thanks, I tried this instead: 

`die "TODO: '$TODO_ACTIONS_DIR' does not exist."`

But still `make test` throws the same error. Perhaps a bug with the completion?

Reply:
Ah, `die()` currently doesn't print to stderr; that's another oversight. I've fixed that in #404; I've you base your patch on that (or just manually add the `>&2` in `die()`), it should work as I said.

Reply:
Thanks for the patch; I seem to be quite a skilled operator ;)

I will base this tomorrow on your PR.